app/models/campaign.rb

```diff
@@ -0,0 +1,15 @@
+class Campaign < ApplicationRecord
+  belongs_to :user
+  validates :title, presence: true,
+                    uniqueness: true,
+                    length: { in: 3..30 },
+                    format: { with: /\A[a-zA-Z0-9\s]+\z/ }
```

Comment:
why do we need this format validation?

Reply:
To check if title doesn't include unknown symbols. Don't need this  validation?

Reply:
does it make sense?